domain/registry/store.go

```diff
@@ -15,10 +15,10 @@ package registry
 
 import (
 	"errors"
+	"github.com/control-center/serviced/datastore/elastic"
```

Comment:
In general you should fix the import order for all files you changed imports. 3rd party imports should be grouped.

Reply:
If you don't mind I'll do full refactoring of imports after all test will be succeeded